File: app/src/main/java/com/pdamkotasmg/happywork/api/google/ApiConfigGoogle.java
package com.pdamkotasmg.happywork.api.google;


import com.pdamkotasmg.happywork.api.server.ApiService;

import retrofit2.Retrofit;
import retrofit2.converter.gson.GsonConverterFactory;

public class ApiConfigGoogle {

    public static ApiService getApiService(){
        Retrofit retrofit = new Retrofit.Builder()
                .baseUrl("https://www.google.com/recaptcha/api/")
                .addConverterFactory(GsonConverterFactory.create())
                .build();

        return retrofit.create(ApiService.class);
    }
}

File: app/src/main/java/com/pdamkotasmg/happywork/api/google/ApiConfigGoogle.kt
package com.pdamkotasmg.happywork.api.google

import com.pdamkotasmg.happywork.api.server.ApiService
import retrofit2.Retrofit
import retrofit2.converter.gson.GsonConverterFactory

object ApiConfigGoogle {
    fun getApiService(): ApiService? {
        val retrofit = Retrofit.Builder()
                .baseUrl("https://www.google.com/recaptcha/api/")
                .addConverterFactory(GsonConverterFactory.create())
                .build()
        return retrofit.create(ApiService::class.java)
    }
}